### DISCUSSION TITLE
Contact form from product page for availability

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#1688] [Questions & Answers]
I am looking for an extension paid or not that will give me the possibility to ask a question about the stock on a product page. Once a button is pressed, a pop-up window will appear containing a form with the **product name**, **its custom options**, **quantity**, a field for the **message** and two buttons for Send and Cancel. Once the Send button is pressed the form is sent to an email address.  Nothing is registered in the database, no special section in Magento Backend, just a basic and efficient functionality for asking a question.

Any help is welcome.

**Comment by m-overlund** [comment]
So you want the customer to be able to choose custom options in the popup?
I would use a popup tool like sleeknote.com and send the product data through Google Tag Manager.

**Reply by addison74** [reply]
I appreciate the help but it is not what I am looking for. In addition it must not be complicated but as simple as possible.

The idea is the following there are special products made to order where the stock information cannot be maintained in real time but it is necessary to discuss with the supplier. A potential customer will click on a [Ask for Stock] link and in the modal window fills in **Name**, **Email** then select the **Options** of the product that interest him such as Color, Size. If he wants to send additional information/requests then he can fill the **Message** field. Then he clicks [Send] and the form reaches the store owner by email.

I have several solutions from which I can start an implementation but I don't have time and I would prefer something ready-made that in case of changes does not last too long. however, I did not find the implementation related to the product options, but I can take it from the product page since the information is there.
